install_analysistoolsts.sh

```diff
@@ -21,7 +21,10 @@ echo -e "Installing AnalysisToolsTS... "
 mkdir -p "${INSTALLATION_DIR}"
 
 # Copy cloned files to AnalysisToolsTS directory
-rsync -rtz "${REPO_DIR}" "${INSTALLATION_DIR}"
+if [ $# -eq 0 ]; then
+    rsync -rtz "${REPO_DIR}/src" "${INSTALLATION_DIR}"
+    rsync -rtz "${REPO_DIR}/CMakeLists.txt" "${INSTALLATION_DIR}"
```

Comment:
rsync allows multiple src arguments ([man docs](https://linux.die.net/man/1/rsync)), and so can rewrite this to be:
```suggestion
    rsync -rtz "${REPO_DIR}/src" "${REPO_DIR}/CMakeLists.txt" "${INSTALLATION_DIR}"
```

Reply:
Thanks for pointing that out. Fixed